test/utils/Arrays.t.sol

```diff
@@ -2,12 +2,30 @@
 
 pragma solidity ^0.8.20;
 
+// solhint-disable func-name-mixedcase
+
 import {Test} from "forge-std/Test.sol";
+import {SymTest} from "halmos-cheatcodes/SymTest.sol";
 import {Arrays} from "@openzeppelin/contracts/utils/Arrays.sol";
 
-contract ArraysTest is Test {
+contract ArraysTest is Test, SymTest {
     function testSort(uint256[] memory values) public {
         Arrays.sort(values);
+        _assertSort(values);
+    }
+
+    function check_Sort() public {
+        uint256[] memory values = new uint256[](3);
```

Comment:
Any reason to use 3 ? If possible we should use something bigger (ideally an arbitrary value - within some bounds)

Reply:
Unfortuanately 3 is the largest value that didn't timeout. The sorting algorithm's path explosion after 4 seems to be too much already, and make sense.

I would even consider removing this check whatsoever.